user-guide/Advanced_Modules/Job_Manager/Accessing_the_jobs_app.md

```diff
@@ -0,0 +1,30 @@
+---
+uid: Accessing_the_jobs_app
+---
+
+# Accessing the Jobs app
+
+To access the Jobs app, depending on your setup, go to one of the following addresses in a browser:
+
+```txt
+http://[DMA]/jobs
+https://[DMA]/jobs
+```
+
+> [!CAUTION]
+> If you use a DataMiner version prior to DataMiner 10.1.7, we strongly advise that you use HTTPS when you use DataMiner client applications over public internet. If you do not do so, all information – including logon credentials – is sent as plain, unencrypted text over the internet. From DataMiner 10.1.7 onwards, client-server communication is encrypted by default. See also: [Setting up HTTPS on a DMA](xref:Setting_up_HTTPS_on_a_DMA).
+
+## Accessing the Jobs app via the DataMiner Cloud Platform
+
+If you have been granted [remote access](xref:Controlling_remote_access) to a cloud-connected DataMiner System as a guest, you can connect as follows:
```

Comment:
i would remove the 'as a guest' in all the web apps pages, the users need a DCP and DataMiner account so they are not really a guest anymore (?)

```suggestion
If you have been granted [remote access](xref:Controlling_remote_access) to a cloud-connected DataMiner System, you can connect as follows:
```